pkg/metrics/metrics.go

```diff
@@ -16,16 +16,47 @@ package metrics
 
 import "go.opencensus.io/stats"
 
+const (
+	// APICallDurationName is the name of API duration metric
+	APICallDurationName = "api_duration_seconds"
```

Comment:
Do we hard code these names in the e2e tests? Or are we using things like `APICallDuration.Name()`?

Reply:
They are matched by the .Name() functions, not the string value.